### DISCUSSION TITLE
Datasets of other demo NeRF scenes | Minimum required images

### DISCUSSION THREAD

**Discussion by jessekirbs** [NVlabs/instant-ngp#404] [Q&A]
Hello!

First of all, amazing work here. This is mindblowing stuff. I'm curious if the datasets of any other demo NeRF scenes are available, specifically the lab with the robotic arm and the person holding the polaroid camera from [this article](https://www.theverge.com/2022/3/25/22996130/nvidia-instant-nerf-demo-ai-render-3d-scenes-2d-photos). Were the people just holding still while the footage/images were taken? What do you think the minimum required images would be to get a good result of a person? Thank you!

**Comment by Tom94** [comment]
Hi there! While these datasets aren't public at the moment, I can confirm that the people were indeed holding still for a minute or so while an iPhone (robot dataset) or a DSLR (polaroid dataset) recorded a video around them. These videos were then ingested with the bundled scripts and unsatisfactory images (blurred or closed eyes) manually removed. Cheers!

**Reply by jessekirbs** [reply]
Good to know. Thanks for the reply, @Tom94!
